tests/operation/testoperation.py

```diff
@@ -57,17 +57,18 @@ def test_image_gradient(self):
     def test_image_contours(self):
         img = Image(np.array([[0, 1, 0], [3, 4, 5.], [6, 7, 8]]))
         op_contours = contours(img, levels=[0.5])
-        contour = Contours([[(-0.5,  0.416667, 0.5), (-0.25, 0.5, 0.5)],
-                             [(0.25, 0.5, 0.5), (0.5, 0.45, 0.5)]],
+        contour = Contours([[(-0.5,  0.416667, 0.5), (-0.25, 0.5, 0.5),
+                             (np.NaN, np.NaN, 0.5), (0.25, 0.5, 0.5),
```

Comment:
I don't quite understand where the NaNs come from...

Reply:
``contours`` now returns a Contours element with one array per level, if there's multiple contours at a level they will be NaN separated.